cpp/src/arrow/vendored/xxhash/README.md

```diff
@@ -17,6 +17,5 @@
   under the License.
 -->
 
-The files in this directory are vendored from xxHash git tag v0.8.0
+The files in this directory are vendored from xxHash git tag v0.8.1
 (https://github.com/Cyan4973/xxHash).
-Includes https://github.com/Cyan4973/xxHash/pull/502 for Solaris compatibility
```

Comment:
https://github.com/Cyan4973/xxHash/commits/v0.8.1?after=35b0373c697b5f160d3db26b1cbb45a0d5ba788c+174&branch=v0.8.1&qualified_name=refs%2Ftags%2Fv0.8.1

This is already included in v0.8.1, so removed this line